src/components/CreditCardForm.tsx

```diff
@@ -53,11 +53,16 @@ export const CreditCardForm = ({ validationErrors, setValidationErrors }: Credit
   }, [payRequest]);
 
   useEffect(() => {
-    const cardTypeName = String(getCardType(number, supportedNetworks as unknown as CardTypeNames[])?.type);
-    if (options?.styleProps?.showSupportedCards === false) {
```

Comment:
made the change here so showSupportedCards only controls card preview img rotation, showCardIcon is used for all other imgs (cardTypes, errorCard, unknownCard, Cvv)